src/components/QandA/QuestionList.jsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 
-const QuestionList = (props) => {
+function QuestionList ({count, questions}) {
 
+  //console.log('questions = ', questions);
+  //console.log('questions.results = ', questions.results);   // default: 4
 
-  // console.log('props = ', props)
-  // console.log('props = ', props);
-  // console.log('props.questions = ', props.questions);
-  // console.log('props.count = ', props.count);   // default: 4
-
-  const last = props.count;
+  const last = count;
```

Comment:
I've found that for all props being passed, it's useful to do a useEffect method and set the props as state if you need to use them in your return DOM elements. So it would probably clear some of your errors if you set both count and questions as state, and then populate them in their own useEffect methods.